configs/pytorch.json

```diff
@@ -1,8 +1,14 @@
 {
   "index_name": "pytorch",
   "start_urls": [
+    "https://pytorch.org",
+    "https://pytorch.org/index.html",
     "https://pytorch.org/get-started/",
-    "https://pytorch.org/get-started/locally/"
+    "https://pytorch.org/get-started/locally/",
+    "https://pytorch.org/features",
+    "https://pytorch.org/ecosystem",
+    "https://pytorch.org/blog/",
```

Comment:
We usually do not feature blog. But since the content of this section is helpful for the community, we will scrap it. We will provide you a specific tag for this part and the tutorials one in order to provide you a way to refine the search.

Reply:
I can remove it for now so we don't add too much complexity in this initial change.